drivers/bridge/bridge.go

```diff
@@ -371,44 +372,52 @@ func (d *driver) DeleteNetwork(nid types.UUID) error {
 	return err
 }
 
-func (d *driver) CreateEndpoint(nid, eid types.UUID, epOptions map[string]interface{}) (*sandbox.Info, error) {
+func (d *driver) CreateEndpoint(nid, eid types.UUID, epInfo driverapi.EndpointInfo, epOptions map[string]interface{}) error {
 	var (
 		ipv6Addr *net.IPNet
 		err      error
 	)
 
+	if epInfo == nil {
+		return errors.New("invalid endpoint info passed")
```

Comment:
We can take care of defining an appropriate error later on in error.go for this.